composer.json

```diff
@@ -0,0 +1,32 @@
+{
+    "name": "beans/beans",
+    "description": "The most innovative theme for WordPress. The real magic is under the hood.",
+    "type": "wordpress-theme",
```

Comment:
This is so that Composer Installers know where to move a theme a to, instead of inside the typical `vendor` directory.